godtools/App/Features/SetupParallel/SetupParallelLanguageView.swift

```diff
@@ -6,19 +6,19 @@
 //  Copyright © 2021 Cru. All rights reserved.
 //
 
-import Foundation
 import UIKit
 
 class SetupParallelLanguageView: UIViewController {
     
     private let viewModel: SetupParallelLanguageViewModelType
     
-    private var closeButton: UIBarButtonItem?
-
+    private let buttonCornerRadius: CGFloat = 6
+    
+    private var tapRecognizer: UITapGestureRecognizer!
+    
     @IBOutlet weak private var animatedView: AnimatedView!
     @IBOutlet weak private var promptLabel: UILabel!
     @IBOutlet weak private var selectLanguageButtonView: UIView!
-    @IBOutlet weak private var selectLanguageButtonGestureRecognizer: UITapGestureRecognizer!
```

Comment:
Hey @reldredge71 instead of having this declared with the forced unwrap ```!``` can you instead make it immutable and allocate this when the view is initialized. I don't think you will be able to set the target and action on initialization of UITapGestureRecognizer, but take a look at the UITapGestureRecognizer addTarget method to do that.

https://developer.apple.com/documentation/uikit/uigesturerecognizer/1624230-addtarget

Reply:
@levieggertcru I'm a little confused.  Are you talking about `selectLanguageButtonView`?  Should it not be an IBOutlet?  Or are you talking about the tapRecognizer?  If it's the tapRecognizer, then I could probably implement it similar to here: https://github.com/CruGlobal/godtools-swift/blob/develop/godtools/App/Services/Renderer/Views/MobileContent/Views/ContentImage/MobileContentImageView.swift#L26

Reply:
Hey @reldredge71 I was talking about the tapGesture.  IBOutlets you have to declare unwrapped ```!```, but anything else it's good practice not to declare with the ```!```.